src/bun.js/api/server.zig

```diff
@@ -1976,7 +1978,7 @@ fn NewRequestContext(comptime ssl_enabled: bool, comptime debug_mode: bool, comp
             if (this.signal) |signal| {
                 this.signal = null;
                 if (this.flags.aborted and !signal.aborted()) {
-                    const reason = JSC.WebCore.AbortSignal.createAbortError(JSC.ZigString.static("The user aborted a request"), &JSC.ZigString.Empty, this.server.globalThis);
+                    const reason = JSC.WebCore.AbortSignal.createAbortError(JSC.ZigString.static("The user aborted a request"), &JSC.ZigString.Empty, globalThis);
```

Comment:
As a follow-up, instead of creating this AbortError, we could probably add an enum to AbortSignal for known types of signals. Then, we can defer creating the JSValue here if it's not used.